view/frontend/templates/customer/account/attachment-list.phtml

```diff
@@ -0,0 +1,55 @@
+<?php
+/** @var $block \LizardMedia\ProductAttachment\Block\Customer\Account\AttachmentList */
+?>
+<?php $attachments = $block->getAttachments() ?>
+<?php if ($attachments->getSize() > 0): ?>
+    <div class="table-wrapper downloadable-products">
+        <table id="my-downloadable-products-table" class="data table table-downloadable-products">
+            <caption class="table-caption"><?= __('My attachments') ?></caption>
+            <thead>
+            <tr>
+                <th scope="col" class="col id"><?= __('Product') ?></th>
+                <th scope="col" class="col title"><?= __('Title') ?></th>
+            </tr>
+            </thead>
+            <tbody>
+            <?php foreach ($attachments as $attachment) : ?>
+                <tr>
+                    <td data-th="<?= $block->escapeHtml(__('Product')) ?>" class="col id">
+                        <strong class="product-name"><?= $block->escapeHtml(
+                                $attachment->getCatalogProductName()
+                                ? $attachment->getCatalogProductName()
+                                : $attachment->getCatalogProductNameDefault()
+                            ); ?>
+                        </strong>
+                    </td>
+                    <td data-th="<?= $block->escapeHtml(__('Title')) ?>" class="col title">
+                        <a href="<?= $block->escapeUrl($block->getDownloadUrl($attachment)) ?>"
+                           title="<?= $block->escapeHtml(__('Start download')) ?>"
+                           class="action download"
+                           <?= $block->getIsOpenInNewWindow() ? 'onclick="this.target=\'_blank\'"' : '' ?>>
```

Comment:
@bartek9007 Maybe move interaction (onclick?) to knockout or external JS? Not necessary, could be.